web-ui/src/main/resources/catalog/components/search/searchmanager/SearchManagerService.js

```diff
@@ -32,7 +32,9 @@
     '$rootScope',
     '$http',
     'gnHttp',
-    function(gnUtilityService, $q, $rootScope, $http, gnHttp) {
+    'Metadata',
+    function(gnUtilityService, $q, $rootScope,
+             $http, gnHttp, Metadata) {
```

Comment:
`Metadata` is not used in this service. You can remove it to avoid unused dependecies.